src/ResourceManager/Storage/Commands.Management.Storage.Test/ScenarioTests/StorageAccountTests.ps1

```diff
@@ -657,3 +661,33 @@ function Test-SetAzureStorageAccountStorageV2
         Clean-ResourceGroup $rgname
     }
 }
+
+<#
+.SYNOPSIS
+Test GetAzureStorageUsage with current Subscription
```

Comment:
Can these tests be run in Record mode by some one in the powershell team? Also, do the tests involve something (infrastructure wise) to exist previously fir them to be recorded successfully?

Reply:
I have record these test and update the PR.

Reply:
Hi blueww thanka for re-recording the tests. Way i wanted to ask in my comment was that can i or some one else on the powershell team re-record these tests using our subscription without any need to provision resources beforehand?

Reply:
You can re-record it when you subscription support "eastus2(stage)" location, and support the new features and new API version.